Packs/Core/ReleaseNotes/3_0_55.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Added the **alter_events_from_decider** argument, use this argument to obtain **events_from_decider** context path as a list (for improved for playbook automation) rather than a dictionary (The original API raw response). 
```

Comment:
```suggestion
- Added the ***alter_events_from_decider*** argument, use this argument to obtain **events_from_decider** context path as a list (for improved for playbook automation) rather than a dictionary (The original API raw response). 
```